Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -295,9 +347,6 @@ - (UserProfile *)localUserProfile
                 DDLogInfo(@"%@ Building local profile.", self.tag);
                 _localUserProfile = [[UserProfile alloc] initWithRecipientId:kLocalProfileUniqueId];
                 _localUserProfile.profileKey = [OWSAES256Key generateRandomKey];
-                [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *_Nonnull transaction) {
-                    [_localUserProfile saveWithTransaction:transaction];
-                }];
                 // Sync local profile to any linked device
```

Comment:
This comment is now misleading - since we're not *just* saving to notify linked devices.

Reply:
Removed.